README.md

```diff
@@ -21,6 +21,9 @@ bookshelf.plugin(require('bookshelf-bcrypt'))
 
 // Enable it on your models
 let User = bookshelf.Model.extend({ tableName: 'users', bcrypt: { field: 'password' } })
+
+// By default, an error will be thrown if a null password is detected. Use the following to allow null passwords
+let User = bookshelf.Model.extend({ tableName: 'users', bcrypt: { field: 'password', allowNullPassword: true } })
```

Comment:
Can you change the name to `allowEmptyPassword`? That makes more sense since password can be `null` or `undefined`